docker-entrypoint.sh

```diff
@@ -1,110 +1,129 @@
 #!/bin/sh
 
-INTEGRITY_CHECK=${HUMHUB_INTEGRITY_CHECK:-1}
```

Comment:
This is mostly a style thing. You are right that, for assignment, the two are identical. I just consider it to be a good practice to double quote as a default behavior whenever there are variables since, in many cases, white space or other unexpected characters can break things in surprising ways sometimes. So, unless I have a specific reason not to, I quote variables.